config/config.go

```diff
@@ -10,10 +10,13 @@ import (
 type Config struct {
 	BindAddr                   string        `envconfig:"BIND_ADDR"`
 	HttpWriteTimeout           time.Duration `envconfig:"HTTP_WRITE_TIMEOUT"`
+	DefaultRequestTimeout      time.Duration `envconfig:"DEFAULT_REQUEST_TIMEOUT"`
```

Comment:
I think line 13 is to generic and should be more specific, like for example:

CantabularRequestTimeout time.Duration `envconfig:"CANTABULAR_REQUEST_TIMEOUT"`